File: 백준/Bronze/9546. 3000번 버스/3000번 버스.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int testCase;
        int k;

        testCase = sc.nextInt();

        for (int i = 0; i < testCase; i++) {
            k = sc.nextInt();

            System.out.println((int) (Math.pow(2, k) - 1));
        }

        sc.close();
    }
}

File: 백준/Bronze/9546. 3000번 버스/3000번 버스.cc
#include <iostream>
#include <cmath>

using namespace std;

int main() {
	int testCase;
	int k;

	cin >> testCase;

	for (int i = 0; i < testCase; i++) {
		cin >> k;

		cout << (int)(pow(2, k) - 1) << "\n";
	}

	return 0;
}